src/drivers/driver_postgres.rs

```diff
@@ -0,0 +1,25 @@
+#![cfg(feature = "driver-postgres")]
+use drivers::Driver as Driv;
+
+pub struct Driver{
+    target: &'static str,
+}
+
+impl Driv for Driver {
```

Comment:
`use drivers::Driver as Driv;` that is the line that makes it understandable...
I simply didn't want the path to the postgres driver to be drivers::driver_postgres::PostgresDriver since it would take mean spelling postgres twice and I tend to spell it with more than one 's' from time to time. I could change it if it is a problem... I could also change the as to something like DriverTrait